test/suite/test_bug018.py

```diff
@@ -74,11 +84,12 @@ def test_close_writefail(self):
 
         # Make a backup for forensics in case something goes wrong.
         backup_dir = 'BACKUP'
-        shutil.rmtree(backup_dir, ignore_errors=True)
-        shutil.copytree('.', backup_dir, lambda src, names: (n for n in names if n != backup_dir))
+        copy_wiredtiger_home('.', backup_dir, True)
```

Comment:
Thanks for this one @sueloverso, I knew `wttest.backup` was wrong because it uses a backup cursor, but forgot about `helper`.